.github/workflows/upload-torch-dynamo-perf-stats.yml

```diff
@@ -0,0 +1,56 @@
+name: Upload torch dynamo performance stats
+
+on:
+  workflow_run:
+    workflows: [inductor, inductor-A100-perf, periodic]
```

Comment:
We only use `inductor-A100-perf` as the data source for the dashboard, unless you are thinking to also upload data for some other purpose.

Reply:
Noted.  I will remove the other twos.  There is no need to store the data if there is no immediate need to use them.  Adding them back is easy enough

Reply:
In fact, this helps address my concern about the number of records to store. `inductor-A100-perf` runs very infrequently, so there will only be few hundred new records per day written to Rockset.